modules/vaos/app/controllers/vaos/preferences_controller.rb

```diff
@@ -2,15 +2,29 @@
 
 module VAOS
   class PreferencesController < VAOS::BaseController
-    def index
-      response = preferences_service.get_preferences(current_user)
+    def show
+      response = preferences_service.get_preferences
+      render json: VAOS::PreferencesSerializer.new(response)
+    end
```

Comment:
This reads to me as if it is returning a collection of `Preference` resources, in which case a `show` seems inappropriate, why not an `index` still?

Reply:
I would characterize Preferences as a singular resource having multiple attributes, not a collection. It's a 1-1 resource belonging to User though.

Reply:
If each attribute returned was part of an array, each attribute having its own ID, then I would agree with you on "collection." But that's not the case. It's a set of attributes (which in spoken language is typically pluralized as "preferences"). Rails makes exceptions for different inflections. But whats important to note is that a User can at most only have 1 set of this resource, hence why it is singular.